diff --git a/src/ffmpeg-1-fixes.patch b/src/ffmpeg-1-fixes.patch
index 5381ddf6..be47f68b 100644
--- a/src/ffmpeg-1-fixes.patch
+++ b/src/ffmpeg-1-fixes.patch
@@ -2,26 +2,6 @@ This file is part of MXE. See LICENSE.md for licensing information.
 
 Contains ad hoc patches for cross building.
 
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Tony Theodore <tonyt@logyst.com>
-Date: Sun, 25 Apr 2021 19:23:19 +1000
-Subject: [PATCH 1/2] fix windres whitespace handling
-
-
-diff --git a/ffbuild/common.mak b/ffbuild/common.mak
-index 1111111..2222222 100644
---- a/ffbuild/common.mak
-+++ b/ffbuild/common.mak
-@@ -82,7 +82,7 @@ COMPILE_NVCC = $(call COMPILE,NVCC)
- 	-$(if $(ASMSTRIPFLAGS), $(STRIP) $(ASMSTRIPFLAGS) $@)
- 
- %.o: %.rc
--	$(WINDRES) $(IFLAGS) --preprocessor "$(DEPWINDRES) -E -xc-header -DRC_INVOKED $(CC_DEPFLAGS)" -o $@ $<
-+	$(WINDRES) $(IFLAGS) --preprocessor "$(DEPWINDRES)" $(addprefix --preprocessor-arg ,-E -xc-header -DRC_INVOKED $(CC_DEPFLAGS)) -o $@ $<
- 
- %.i: %.c
- 	$(CC) $(CCFLAGS) $(CC_E) $<
-
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Mark Brand <mabrand@mabrand.nl>
 Date: Mon, 4 Apr 2022 00:33:50 +0200
diff --git a/src/ffmpeg.mk b/src/ffmpeg.mk
index b486d859..d1d7013d 100644
--- a/src/ffmpeg.mk
+++ b/src/ffmpeg.mk
@@ -3,8 +3,8 @@
 PKG             := ffmpeg
 $(PKG)_WEBSITE  := https://ffmpeg.org/
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 4.2.3
-$(PKG)_CHECKSUM := 217eb211c33303b37c5521a5abe1f0140854d6810c6a6ee399456cc96356795e
+$(PKG)_VERSION  := 5.1.2
+$(PKG)_CHECKSUM := 39a0bcc8d98549f16c570624678246a6ac736c066cebdb409f9502e915b22f2b
 $(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
 $(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.bz2
 $(PKG)_URL      := https://ffmpeg.org/releases/$($(PKG)_FILE)
@@ -40,28 +40,26 @@ define $(PKG)_BUILD
         --disable-pthreads \
         --enable-w32threads \
         --disable-doc \
-        --enable-avresample \
+        --disable-programs \
+        --disable-avdevice \
+        --disable-swresample \
+        --disable-swscale \
+        --disable-postproc \
+        --disable-avfilter \
+        --disable-encoders \
+        --disable-muxers \
+        --disable-devices \
+        --disable-filters \
         --enable-gpl \
         --enable-version3 \
         --extra-libs='-mconsole' \
-        --enable-avisynth \
-        --enable-gnutls \
-        --enable-libass \
         --enable-libbluray \
-        --enable-libbs2b \
-        --enable-libcaca \
-        --enable-libmp3lame \
         --enable-libopencore-amrnb \
         --enable-libopencore-amrwb \
         --enable-libopus \
         --enable-libspeex \
-        --enable-libtheora \
-        --enable-libvidstab \
-        --enable-libvo-amrwbenc \
         --enable-libvorbis \
         --enable-libvpx \
-        --enable-libx264 \
-        --enable-libxvid \
         --extra-ldflags="-fstack-protector" \
         $($(PKG)_CONFIGURE_OPTS)
     $(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)'
