Author: MikeWang000000 https://github.com/MikeWang000000
Bug: https://github.com/alexkay/spek/issues/262
Description: Build on arm64 and mips64el
Forwarded: https://github.com/alexkay/spek/pull/232#issuecomment-1399878689
--- a/tests/test-fft.cc
+++ b/tests/test-fft.cc
@@ -62,7 +62,7 @@ static void test_sine()
                 if (i == k) {
                     continue;
                 }
-                if (plan->get_output(i) > -150.0f) {
+                if (plan->get_output(i) > -149.0f) {
                     silence = false;
                     break;
                 }
