Author: Colin Watson <cjwatson@ubuntu.com>
Description: Fix ActionEntry ambiguity resulting in Vala build error
Bug-Debian: http://bugs.debian.org/638456

--- a/src/spek-window.vala
+++ b/src/spek-window.vala
@@ -31,7 +31,7 @@
 		private FileFilter filter_audio;
 		private FileFilter filter_png;
 
-		private const ActionEntry[] ACTION_ENTRIES = {
+		private const Gtk.ActionEntry[] ACTION_ENTRIES = {
 			{ "File", null, N_("_File") },
 			{ "FileOpen", STOCK_OPEN, null, null, null, on_file_open },
 			{ "FileSave", STOCK_SAVE, null, null, null, on_file_save },
